x-team/xp-ui
# Fix non-existing avatar loading

**Release Type:** *Bug Fix* 


Fixes https://x-team-internal.atlassian.net/browse/XP-3747

## Description

Fixes the case when we do not pass an avatar to the `AccountConfirmation` UI component which results in `404` HTTP error as shown below:

![image](https://user-images.githubusercontent.com/579331/81403380-ee940f00-913b-11ea-8d59-1fb4be0fe0f8.png)

## Checklist

**Before submitting a Pull Request,** please make sure the following is done:



## Steps to Test or Reproduce

I've added a new story to test the fix. Try to revert the change inside `AccountConfirmation.js` component and check the new `avatar src is not passed in`. You should see the aforementioned error. With the fix in place, you just see a default avatar based on the user name (based on the `alt` prop value) as it's expected and no `404` errors anymore:

![image](https://user-images.githubusercontent.com/579331/81403525-374bc800-913c-11ea-9273-6846c479a777.png)

## Impacted Areas in Application

- `AccountConfirmation` UI component is adjusted
- `Avatar.storirs.js` are updated to use the proper story name notation

## Screenshots

![image](https://user-images.githubusercontent.com/579331/81403525-374bc800-913c-11ea-9273-6846c479a777.png)
